### ISSUE TITLE
fusibile error

### ISSUE THREAD

**Issue by JadeWeigh** [maybeLx/MVSFormerPlusPlus#32]
fusibile was successfully made, but the following error occurred when running test.py.
`Run depth map fusion & filter
./fusibile -input_folder ./output/dtu1115/_1536x1152/scan1/points_mvsnet/ -p_folder ./output/dtu1115/_1536x1152/scan1/points_mvsnet/cams/ -images_folder ./output/dtu1115/_1536x1152/scan1/points_mvsnet/images/ --depth_min=0.001 --depth_max=100000 --normal_thresh=360 --disp_thresh=0.1 --num_consistent=2.0 -color_processing
Segmentation fault (core dumped)`

**Comment by ewrfcas** [comment]
Tips: You should revise CUDA_NVCC_FLAGS in CMakeLists.txt according the gpu device you used. We set -gencode arch=compute_70,code=sm_70 instead of -gencode arch=compute_60,code=sm_60 with V100 GPUs. For other GPU types, you can follow

# 1080Ti
-gencode arch=compute_60,code=sm_60

# 2080Ti
-gencode arch=compute_75,code=sm_75

# 3090Ti
-gencode arch=compute_86,code=sm_86

# V100
-gencode arch=compute_70,code=sm_70
More compile relations could be found in [here](https://arnon.dk/matching-sm-architectures-arch-and-gencode-for-various-nvidia-cards/)

**Comment by JadeWeigh** [comment]
Yeah, I have already modified the `CUDA_NVCC_FLAGS`. My GPU is 3090, so I changed it to `-gencode arch=compute_86,code=sm_86`, but the error still occurs, and I don't know how to resolve it.

**Comment by JX-Github-web** [comment]
![image](https://github.com/user-attachments/assets/112c995e-e08b-469e-911e-4491731acf4d)
I'm using a 2080Ti and configured it according to this requirement, 
but it appears: ”sh: 1: ./fusibile: Permission denied“

**Comment by JX-Github-web** [comment]
Hello, as per your suggestion, I tried, but still get an error
sh: 1: ./fusibile: Permission denied

**Comment by JX-Github-web** [comment]
Hello, thank you, follow your advice, I tried it, this one is useful.
--fusibile_exe_path ./fusibile/fusibile
